airbyte-integrations/connectors/source-airtable/source_airtable/spec.json

```diff
@@ -9,18 +9,21 @@
     "properties": {
       "api_key": {
         "type": "string",
-        "description": "The API key for the Airtable account. https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-",
+        "description": "The API Key for the Airtable account. See the <a href=\"https://support.airtable.com/hc/en-us/articles/219046777-How-do-I-get-my-API-key-\">Support Guide</a> for more information on how to obtain this key.",
+        "title": "API Key",
         "airbyte_secret": true,
         "examples": ["key1234567890"]
       },
       "base_id": {
         "type": "string",
-        "description": "The base ID to integrate data from",
+        "description": "The Base ID to integrate the data from",
+        "title": "Base ID",
```

Comment:
any info on where to get this base ID from?

Reply:
A will add the description: "You can find the Base ID following the link <a href=\"https://airtable.com/api\">Airtable API</a>, log into your account, select the base you need and find Base ID in the docs."